pandora/models/pandora.py

```diff
@@ -223,11 +226,26 @@ class AdItem(PlaylistModel):
     def is_ad(self):
         return True
 
-    def register_ad(self, station_id):
-        self._api_client.register_ad(station_id, self.tracking_tokens)
+    def register_ad(self, station_id=None):
+        if not station_id:
+            station_id = self.station_id
+        if self.tracking_tokens:
+            self._api_client.register_ad(station_id, self.tracking_tokens)
+        else:
+            raise ParameterMissing('No ad tracking tokens provided for '
```

Comment:
Is there a point in raising `ParameterMissing` here? From your description it sounds like this is a common occurrence in some regions and totally under Pandora's control. I vote we just play the ad but don't register it.